docs/core/tutorials/upgrade.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Upgrade to a new .NET version
+description: Learn how to upgrade an app to a new .NET version.
+ms.topic: how-to
+ms.date: 03/25/2024
+---
+# Upgrade to a new .NET version
+
+New .NET versions are [released each year](https://github.com/dotnet/core/blob/main/releases.md). Many developers start the upgrade process as soon as the new version is available, while others wait until the version they are using is no longer supported. The upgrade process has multiple aspects to consider.
+
+## Requesting an upgrade (as a user)
+
+You may need an app to use a new .NET version and to make a rquest to the team or vendor that produced the app.
+
+Common reasons to upgrade to a new .NET version:
+
+- The currently used .NET version is no longer supported
+- The new version supports a new operating system
+- The new version has an important performance or security feature

```

Comment:
Also has a new API or "feature" in general?

Reply:
I was writing that from the point of an end-user. They are less likely to know about an API. Perhaps I should just delete/re-frame that whole section and then include a longer description of "why upgrade". What do you think?